packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -413,7 +413,7 @@ export const Tree = (): ReactElement => {
                     onContextMenu={(e) => isExtensionPoint && handleOpenTooltip(e, tooltipId)}>
                     {groupHeaderProps?.group?.count !== 0 && (
                         <Icon
-                            className={`${chevronTransform}`}
+                            className={`${chevronTransform} active-foreground`}
```

Comment:
I don't think we should add new class for this. We already have a class for selected row that we can use `app-panel-selected-bg` in combination with the row class `tree-row`. All icons in the tree should be handled generically and we shouldn't need to fix  this for each of them, but with current solution extension icons when selected still has wrong colour.

![image](https://github.com/user-attachments/assets/cf7227d6-29db-44e9-b3f7-922e953e91ef)